packages/ember-testing/tests/helpers_test.js

```diff
@@ -355,27 +350,29 @@ test('`click` triggers appropriate events in order', function() {
 
   var click, wait, events;
 
-  App.IndexView = EmberView.extend({
-    classNames: 'index-view',
+  App.register('component:click-test-helper', Component.extend({
+    classNames: 'click-test-helper',
 
     didInsertElement() {
       this.$().on('mousedown focusin mouseup click', function(e) {
         events.push(e.type);
       });
     },
 
-    Checkbox: Checkbox.extend({
-      click() {
+    actions: {
+      checkboxClicked() {
```

Comment:
@nickiaconis this seemed to be asserting click behaviors. Per the description:

```js
'`click` triggers appropriate events in order'
```

However by using actions instead of a component with a `click` and `change` handler, I think the intended timing test was lost (now it tests action timing). Can this be tweaked to use a component with `click` and `change` where `{{view view.Checkbox}}` was before?